resources/ext/cli/ruby.erb

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env bash
 
+cli_defaults=${INSTALL_DIR}/cli/cli-defaults.sh
+
+CLASSPATH=${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %>
+
+if [ -e "$cli_defaults" ]; then
+  . $cli_defaults
+  if [ $? -ne 0 ]; then
+    echo "Unable to initialize cli defaults, failing irb subcommand." 1>&2
+    exit 1
+  fi
+fi
+
```

Comment:
Is there any way we could abstract this somehow so its not copied betwen all three of these files? Or would that be more complicated than it's worth?